crates/flux-refineck/src/fixpoint_encoding.rs

```diff
@@ -565,6 +595,59 @@ where
     fn def_span(&self) -> Span {
         self.genv.tcx.def_span(self.def_id)
     }
+
+    fn alias_pred_sort(arity: usize) -> rty::PolyFuncSort {
+        let mut sorts = vec![];
+        for i in 0..arity {
+            sorts.push(rty::Sort::Var(rty::SortVar::from(i)));
+        }
+        sorts.push(rty::Sort::Bool);
+        rty::PolyFuncSort::new(arity, rty::FuncSort { inputs_and_output: List::from_vec(sorts) })
+    }
+
+    fn alias_pred_gen(&mut self, arity: usize) -> AliasPredInfo {
+        let n = self.alias_preds.len();
+        let sym = Symbol::intern(&format!("alias_pred_{n:?}"));
```

Comment:
`sym` is only used to print a comment next to the constant declaration in the `smt2` constraint, so we could just pretty print the alias pred (or even simpler just print it's an alias pred).

Reply:
hmm looks like its used here as an actual symbol?
https://github.com/flux-rs/flux/blob/7948b81601ea8da8795dcc0115d11a5ebc494efc/crates/flux-refineck/src/fixpoint_encoding.rs#L606

Reply:
(For now, just printing the `alias_pred` as you suggest.)